docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -264,23 +257,23 @@ subtasks which are invoked in sequence:
 Once all of these commands have completed there are a couple of final tasks left that
 aren't strictly able to be automated:
 
-* If this is a stable release, it will be necessary to perform a post-release merge
```

Comment:
we should post release merge all our releases, we want RC logs be published on docs.silverstripe.org, don't we?